### PR TITLE
Czech translations

### PR DESCRIPTION
**Pull request by LubosKolouch**

### CODE REVIEW

File: call_your_mom/locale/cs_CZ/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+
+#: templates/call_your_mom/ack_task.html:8
+#, python-format
+msgid "Task done: %(task_name)s"
+msgstr "Úkol splněn: %(task_name)s"
+
+#: templates/call_your_mom/ack_task.html:14
+msgctxt "task form"
+msgid "Done on:"
+msgstr "Splněno :"
```
Comment (remram44) [inline call_your_mom/locale/cs_CZ/LC_MESSAGES/django.po]: Does Czech put spaces before colons `:` like French?

**Review by remram44 — COMMENTED** [review]
Thank you so much @LubosKolouch! I was not expecting any contributions here :blush:

I'm wondering if there might be some inconsistency in the spacing before colons but maybe there's a more complicated rule (I don't know anything about Czech).

**Comment by LubosKolouch** [comment]
Hi I checked the rules and there should be no space before the : I will change it and resubmit..

**Comment by LubosKolouch** [comment]
Added commit with the fixes...

**Comment by remram44** [comment]
Thanks again!

**Pull request merged**